`DispersiveMultiLayer` no longer uses MATLAB (#286)
UCL/TDMS: doc/developers.md

To run the unit tests, [compile](#compiling) with `-DBUILD_TESTING=ON`. Then run `ctest` from the build directory or execute the test executable `./tdms_tests`.

It's good practice, and reassuring for your pull-request reviewers, if new C++ functionality is covered by unit tests.

#### Benchmark Scripts and Data Generation

The `tdms/tests/unit/benchmarking` directory contains scripts that produce input data for the unit tests, or that provide benchmarks for the units that are tested.

A number of unit tests require the presence of a `.mat` or `.hdf5` file to read/write data from/to during testing.

> The locations of these files are coded into `tests/include/unit_test_utils.h`, but the files themselves are not committed to the repository.
> You can create the data files locally by running the `setup_unit_tests.m` and `create_hdf5_data.py` scripts locally when you wish to run the unit tests - the CI does this automatically on GitHub.

Author: ```suggestion
The locations of these files are coded into `tests/include/unit_test_utils.h`, but the files themselves are not committed to the repository - they can be created by running the `setup_unit_tests.m` and `create_hdf5_data.py` scripts.
```
